base/compiler/optimize.jl

```diff
@@ -374,33 +380,19 @@ isknowntype(@nospecialize T) = (T == Union{}) || isconcretetype(T)
 
 function statement_cost(ex::Expr, line::Int, src::CodeInfo, spvals::SimpleVector, slottypes::Vector{Any}, params::Params)
     head = ex.head
-    if is_meta_expr_head(head) || head == :copyast # not sure if copyast is right
+    if is_meta_expr_head(head)
         return 0
-    end
-    argcost = 0
-    for a in ex.args
```

Comment:
What's the motivation for making this non-recursive?

Reply:
The structure of the IR has changed such that calls no longer contain other calls, so it's just unnecessary.